lib/presentation/widget/destination_picker/destination_picker_viewmodel.dart

```diff
@@ -47,7 +47,7 @@ import 'package:rxdart/rxdart.dart';
 class DestinationPickerViewModel extends BaseViewModel {
   final GetAllSharedSpacesInteractor _getAllSharedSpacesInteractor;
   final AppNavigation _appNavigation;
-  final BehaviorSubject<WorkGroupNodesSurfingArguments> currentNodeObservable = BehaviorSubject.seeded(null);
+  final BehaviorSubject<SharedSpaceDocumentArguments> currentNodeObservable = BehaviorSubject.seeded(null);
```

Comment:
Can you explain to me why is a `BehaviorSubject`? why not other data type?

Reply:
It serves as a StreamController to listen for changes to the 'SharedSpaceDocumentArguments' parameter to update the DestinationPicker screen widget.